graphics/macgui/macfontmanager.cpp

```diff
@@ -640,6 +648,41 @@ int MacFontManager::registerFontName(Common::String name, int preferredId) {
 	return id;
 }
 
+int MacFontManager::registerTTFFont(Graphics::TTFMap *ttfList) {
+	int ret = 1;
+
+	for (int i = 0; i < 4; i++) {
+		int id = 1000;
+		Common::String name = ttfList[i].ttfName;
+
+		if (name.empty()) {
+			if (ret == 1)
+				ret = id;
+			continue;
+		}
+
+		if (_fontIds.contains(name)){
```

Comment:
Why is it? What if I want to specify the same font twice on purpose?

Also, the code formatting here is not correct.

Reply:
deleted the warning